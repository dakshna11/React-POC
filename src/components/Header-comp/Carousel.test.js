import React from "react";
import { shallow } from "enzyme";
import Carousel from "./Carousel";

const carousel = 'Test Carousel'
let wrapped = shallow(<Carousel>{carousel}</Carousel>)

describe('Carousel', ()=>{
    it('should render the Carousel Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })
})