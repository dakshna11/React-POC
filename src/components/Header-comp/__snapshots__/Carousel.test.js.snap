// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel should render the Carousel Component correctly 1`] = `
<Carousel
  axis="horizontal"
  centerSlidePercentage={80}
  interval={3000}
  labels={
    Object {
      "item": "slide item",
      "leftArrow": "previous slide / item",
      "rightArrow": "next slide / item",
    }
  }
  onChange={[Function]}
  onClickItem={[Function]}
  onClickThumb={[Function]}
  onSwipeEnd={[Function]}
  onSwipeMove={[Function]}
  onSwipeStart={[Function]}
  preventMovementUntilSwipeScrollTolerance={false}
  renderArrowNext={[Function]}
  renderArrowPrev={[Function]}
  renderIndicator={[Function]}
  renderItem={[Function]}
  renderThumbs={[Function]}
  selectedItem={0}
  showArrows={true}
  showIndicators={true}
  showStatus={true}
  showThumbs={true}
  statusFormatter={[Function]}
  stopOnHover={true}
  swipeScrollTolerance={5}
  swipeable={true}
  transitionTime={350}
  verticalSwipe="standard"
  width="100%"
>
  <div>
    <img
      src="carousel-1.jpg"
    />
    <p>
      Image 1
    </p>
    <p
      className="legend"
    >
      Legend 1
    </p>
  </div>
  <div>
    <img
      src="carousel-2.jpg"
    />
    <p
      className="legend"
    >
      Legend 2
    </p>
  </div>
  <div>
    <img
      src="carousel-3.jpg"
    />
    <p
      className="legend"
    >
      Legend 3
    </p>
  </div>
</Carousel>
`;
